### PR TITLE
[DEV APPROVED] 10486 hide chat popup

### PR DESCRIPTION
**Pull request by andreferreiradlw**
[TP 10486](https://moneyadviceservice.tpondemand.com/restui/board.aspx?#page=board/5253959470622112123&appConfig=eyJhY2lkIjoiNTM0ODcyOThDQjBDMTMyMDgyMTBENEZEM0QwM0ZBQzkifQ==&boardPopup=userstory/10486/silent)

This is the final PR for the new WhatsApp Popup [fixture](https://moneyadviceservice.tpondemand.com/restui/board.aspx?#page=board/5253959470622112123&appConfig=eyJhY2lkIjoiNTM0ODcyOThDQjBDMTMyMDgyMTBENEZEM0QwM0ZBQzkifQ==&boardPopup=feature/10483).

Automatically hide the popup once the user begins to scroll and provide them with the option to show the button again.

### CODE REVIEW

File: app/views/shared/_contact_panels.html.erb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <%= render 'shared/pas_chat_banner' if pensions_and_retirement_page? %>
 
 <div class="l-constrained">
-  <div class="l-contact-panels t-contact-panels <%= 'l-contact-panels--no-border-top' if contact_panels_border_top %> <%= 'l-contact-panels--homepage' if contact_panels_homepage %>">
+  <div class="l-contact-panels t-contact-panels <%= 'l-contact-panels--no-border-top' if contact_panels_border_top %> <%= 'l-contact-panels--homepage' if contact_panels_homepage %>" data-dough-contact-panels>
```
Comment (margOnline) [inline app/views/shared/_contact_panels.html.erb]: This markup really needs to be refactored. The erb code can be extracted abstracted into the presenter, happy to help with this refactoring. I understand that this markup was written previously and strictly speaking not part of this pr; but your changes highlight its need for refactoring.

Comment (andreferreiradlw) [reply app/views/shared/_contact_panels.html.erb]: I agree but I'm not sure if I should be the one to do it, I've added this markup to use it as a selector in javascript. These panels are also going to be redone in ticket 10511.

Comment (margOnline) [reply app/views/shared/_contact_panels.html.erb]: @andreferreiradlw I'll have a look to see how easy it is (or not)

Comment (margOnline) [reply app/views/shared/_contact_panels.html.erb]: @andreferreiradlw I think pr #2126 is a small improvement, wdyt?

Comment (andreferreiradlw) [reply app/views/shared/_contact_panels.html.erb]: @margOnline it looks better then before in my opinion :)
